dev/src/external-modules.d.ts

```diff
@@ -16,3 +16,4 @@
 
 // TODO(mrschmidt): Come up with actual definitions for these modules.
 declare module 'functional-red-black-tree';
+declare module 'length-prefixed-json-stream';
```

Comment:
I see what you did there, adding work to my TODO :)

Reply:
Damn, I should have picked someone else.